templates/optimizers.yaml

```diff
@@ -123,3 +123,30 @@ spec:
         path: /app/optimization-results.json
       - name: optimized-parameters
         path: /app/optimized_parameters.json
+
+
+  # CMA-ES optimization returns `xbest`, the best solution evaluated, but one might want to have `xfavorite`, which is the current best estimate of the optimum
+  - name: extract-xfavorite-parameters-from-cmaes-opt-results
+    parent: generic-task
+    inputs:
+      parameters:
+      - name: command
+        value: python3 main_script.py
+      artifacts:
+      - name: optimization-results
+        path: /app/optimization-results.json
+      - name: main-script
+        path: /app/main_script.py
+        raw:
+          data: |
+            from zquantum.optimizers.utils import load_optimization_results
+            from zquantum.core.circuit import save_circuit_template_params
+            import numpy as np
+
+            opt_results=load_optimization_results("optimization-results.json")
+
+            save_circuit_template_params(np.array(opt_results.cma_xfavorite), "optimized-parameters.json")
+    outputs:
+      artifacts:
+      - name: optimized-parameters
+        path: /app/optimized-parameters.json
```

Comment:
Maybe rename this to `favorite-parameters`? But besides that I think this is great!

Reply:
Done 👍